xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-test/xwiki-platform-notifications-test-tests/src/test/it/org/xwiki/platform/notifications/test/ui/NotificationsTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.platform.notifications.test.ui;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.Test;
+import org.xwiki.platform.notifications.test.po.NotificationsTrayPage;
+import org.xwiki.platform.notifications.test.po.NotificationsUserProfilePage;
+import org.xwiki.test.ui.AbstractTest;
+
+/**
+ * Perform tests on the notifications module.
+ * 
+ * @version $Id$
+ * @since 9.4RC1
+ */
+public class NotificationsTest extends AbstractTest
+{
+    private static final String FIRST_USER_NAME = "user1";
+    private static final String SECOND_USER_NAME = "user2";
+
+    private static final String FIRST_USER_PASSWORD = "notificationsUser1";
+    private static final String SECOND_USER_PASSWORD = "notificationsUser2";
+
+    // Number of pages that have to be created in order for the notifications badge to show «X+»
+    private static final int PAGES_TOP_CREATION_COUNT = 21;
+
+    private void setUpUsers() throws Exception
+    {
+        // Remove any previous registered users
+        getUtil().rest().deletePage("XWiki", FIRST_USER_NAME);
```

Comment:
Fixed in 980d620

Documentation fixed in http://dev.xwiki.org/xwiki/bin/view/Community/Testing?viewer=changes&rev1=94.1&rev2=95.1